src/main/cpp/t04_students.cpp

```diff
@@ -29,10 +29,32 @@
 
 #include "t04_students.h"
 #include <iostream>
-
+#include<string>
+#include<set>
+#include<map>
+#include<vector>
 using namespace std;
 
 int t04_students() {
-    
+int n;
+double numb, m1, m2, m3;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'numb' can be reduced.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4562198)